### PR TITLE
Creation Curl

### PR DESCRIPTION
**Pull request by papajhay**

### CODE REVIEW

File: src/Helper/Curl.php [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+		return $this;
+	}
+
+
+	/**
+	 * Destructeur, clot la connection curl
+	 */
+	public function __destruct()
+	{
+		curl_close($this->curlHandle, $this->_connection);
```
Comment (Mokolos) [inline src/Helper/Curl.php:518]: tsy hilaina intsony (sady jereo ihan ny asan le destruct)

File: src/Helper/Curl.php [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+	 */
+	public function getErrorMessage()
+	{
+		//  return curl_error($this->_connection);
+		try {
+			$response = $this->client->request('POST', $this->url);
+			$this->error = $response->getInfo('error');
+		} catch (TransportExceptionInterface $e) {
+			 echo $e->getMessage();
+		}
```
Comment (Mokolos) [inline src/Helper/Curl.php:269]: tokon mis return

File: src/Helper/Curl.php [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+use Symfony\Component\HttpClient\CurlHttpClient;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+
+class Curl
+{
+    /**
+	 * Curl
+	 * @var resource curl
+	 */
+	 protected $_connection = null;
```
Comment (Mokolos) [inline src/Helper/Curl.php:15]: tokon tsy hiasa intsony ny _connection

File: src/Helper/Curl.php [hunk 4]
```diff
@@ -0,0 +1,520 @@
...
+		// on met à jour le paramétre POST et POSTFIELDS
+		$this->curlOptions[CURLOPT_POST] = $enable;
+		$this->curlOptions[CURLOPT_POSTFIELDS] =  $data;
+
+		// si on a désactivé le post
+		if($enable == 0)
+		{
+			// on repasse en méthode get
+			$this->curlOptions[CURLOPT_HTTPGET] = TRUE;
+		}
```
[the remark this answers is not in the document]
Comment (papajhay) [reply src/Helper/Curl.php:422]: tsy azoko ty  $this->method lasa GET

**Pull request merged**